build.gradle

```diff
@@ -163,6 +162,41 @@ subprojects {
     }
 }
 
+evaluationDependsOnChildren()
+
+task dokka(type: org.jetbrains.dokka.gradle.DokkaAndroidTask, overwrite: true) {
```

Comment:
I think we need to update .taskcluster.yml and generate_api-reference_release.sh to use the new task name?
* https://github.com/mozilla-mobile/android-components/blob/master/automation/docs/generate_api-reference_release.sh
* https://github.com/mozilla-mobile/android-components/blob/master/.taskcluster.yml#L70

Maybe we want to remove it from the release task? It's currently a manual process we invoke **before** the release.

Reply:
Renamed our new tasks to `docs`. So, it's the same as before.